tagtog.py

```diff
@@ -44,8 +44,15 @@ def parse_arguments(argv=[]):
     parser = argparse.ArgumentParser(prog="tagtog", description=__doc__, formatter_class=RawTextHelpFormatter)
     subparsers = parser.add_subparsers(dest="action")
 
-    def str2bool(x):
-        return True if x.lower() in {'yes', 'true', 't', 'y', '1'} else False
+    def parse_verify_ssl_argument(x):
+        if os.path.isfile(x):
```

Comment:
My bad, you were right @waldemar-tagtog 

The path can be generalized to be either a single file or a directory of trusted certificates ([Requests documentation here](https://requests.readthedocs.io/en/master/user/advanced/#ssl-cert-verification))

Therefore, let's just check if the path exists at all:

```suggestion
        if os.path.exists(x):
```